### PR TITLE
made changes based on figma to events page

### LINKED ISSUE #1115: Standardize Events Page - Update Horizontal Cards and Mobile SCSS
### Overview
The horizontal cards and some of the mobile scss in the Events page needs to be standardized to match other pages on the site

### Action Items
- [ ] Match horizontal cards and mobile scss of Events page with specs listed on the Figma file (in pink rectangle)

### Resources/Instructions
[Figma Events page file](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=62%3A94) (follow the instructions in the pink rectangle in bottom right)
[Events page](https://www.hackforla.org/events)

The instructions look like this:
Screen Shot 2021-02-26 at 10 40 47 AM

#1041

### PR DESCRIPTION
**Pull request by thatguyish**
fixes #1115 

hackforla
hackforlapt2

### CODE REVIEW

File: _sass/components/_events.scss [hunk 1]
```diff
@@ -100,6 +100,16 @@
   }
 
 }
+@media only screen and (max-width:750px){
+  .mobile-font-verison{
```
Comment (daniellex0) [inline _sass/components/_events.scss]: Fix type-o: "version" instead of verison

**Review by daniellex0 — CHANGES_REQUESTED** [review]
Looks great, and so close- the mobile font is still showing up as 12px, but I noticed it is probably because of the "verison" (version) type-o. After that it'll be ready!

**Review by akibrhast — CHANGES_REQUESTED** [review]
Please fix merge conflicts @thatguyish

**Comment by thatguyish** [comment]
Changed the type-o on the events.scss page and no merge conflicts when commited.

**Comment by akibrhast** [comment]
@thatguyish  please resolve merge conflicts

**Pull request closed without merging**